CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -538,6 +538,11 @@ This will decode whether the user should have access to direct deletion of eleme
 
 For nodes within ways, the node will remain and tags will be removed.
 
+## `hint` & `hintImages` (info button)
+
+When there are doubts on the question itself (terms used, etc.), how to answer or how to find the answer, more information can be provided via info button text & images.
+Examples for such info button would be explanation of what is halal, what is [tactile paving](https://github.com/streetcomplete/StreetComplete/blob/46a3605c1337f88d0d62a2e140121d1e6f9ffd03/app/src/main/java/de/westnordost/streetcomplete/quests/tactile_paving/AddTactilePavingBusStop.kt#L33-L38), what is a sanitary dump station, when a level counts as a roof level and when as a normal level, explanation of the street side puzzle UI, etc.
```

Comment:
Here you link a specific repo version, but everywhere else, the current version of the file is linked.